packages/forms/README.md

```diff
@@ -0,0 +1,37 @@
+# @acpaas-ui/ngx-components/forms
+
+The forms module is a collection of several forms-related modules, like an autocomplete, a datepicker, a mask, a range slider, a timepicker, etc.
+
+## Usage
+
+```typescript
+import {
```

Comment:
I'm not quite sure why you added this piece of documentation.

Reply:
Because it's inline with the `layout` README file.